### PR TITLE
Add eslint testing

### PR DESCRIPTION
**Pull request by skjnldsv**
Time to have proper eslint checks

### CODE REVIEW

**Review by PVince81 — APPROVED** [review]
:+1: didn't know these were missing on server

**Comment by skjnldsv** [comment]
Not working properly, fixing...

File: .github/workflows/lint.yml [hunk 1]
```diff
@@ -38,3 +38,25 @@ jobs:
         composer run cs:check -- --format=checkstyle | cs2pr
         composer run cs:check || ( echo 'Please run `composer run cs:fix` to format your code' && exit 1 )
       shell: bash
+
+  node:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [12.x]
```
Comment (ChristophWurst) [inline .github/workflows/lint.yml:47]: why do we actually still use such an old version? does that have any advantages?

**Review by ChristophWurst — APPROVED** [review]

File: .github/workflows/lint.yml (hunk 1 above)
Comment (skjnldsv) [reply .github/workflows/lint.yml:47]: None, it was the previous lts.

https://github.com/actions/setup-node/issues/26

**Comment by skjnldsv** [comment]
Done!

**Review by marcoambrosini — APPROVED** [review]

**Comment by skjnldsv** [comment]
https://github.com/nextcloud-gmbh/release-script/issues/21

**Review by MorrisJobke — APPROVED** [review]

**Pull request merged**